src/test-gui/java/de/qabel/desktop/ui/accounting/item/AccountingItemPage.java

```diff
@@ -28,4 +23,14 @@ public void select() {
         clickOn("#selectedRadio");
         waitUntil(controller.selectedRadio::isSelected);
     }
+
+    public void openMenu() {
+        clickOn("#menu");
+    }
+
+    public void edit() {
+        clickOn("#menu");
+        controller.identityMenuController.setAlias("new alias identity");
+        assertEquals("new alias identity", controller.getIdentity().getAlias());
```

Comment:
That's not the purpose of a ui test. Of course you set the alias but the GUI tests are there to verify that you can actually do that by interacting with the user interface. You should click here in the edit button and then insert the new alias into the textfield of the input dialog. (like the old `edit()` method did)